bin/impl/util.sh

```diff
@@ -161,9 +161,9 @@ Possible components:
     fluo-yarn  $2 Apache Fluo YARN and its dependencies (Fluo, Accumulo, Hadoop, & ZooKeeper)
     zookeeper  $2 Apache ZooKeeper
 
-Options:
+Options (Only work for fluo & accumulo components):
```

Comment:
```suggestion
Options (Only works for fluo & accumulo components):
```

Reply:
I read it as "Options only work for fluo and accumulo"

Reply:
I wouldn't read a parenthetical that is capitalized as a continuation of a previous statement. Maybe if it were lower-case and used square brackets, I would, but not like this. Maybe:

```suggestion
Options (these only work for fluo and accumulo components):
```